### PR TITLE
RMQ config secrets ignored by git

### PR DESCRIPTION
**Pull request by jacobmiller22**
As mentioned in #24 , the RabbitMQ secrets should be ignored by git as they are specific to a developers device.

### CODE REVIEW

File: .github/workflows/codecov.yml [hunk 1]
```diff
@@ -47,11 +43,7 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v3
       - name: Start RabbitMQ server
-        run:  |
-          sudo chown -R 1001:root rabbitmq_conf/server
-          sudo chmod 755 rabbitmq_conf/server
-          sudo chmod 400 rabbitmq_conf/server/*
-          docker-compose up -d    
+        run: docker-compose up -d
```
Comment (gftea) [inline .github/workflows/codecov.yml:46]: you can call the updated script like below
```
        run: ./start_rabbitmq.sh
```

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -50,55 +48,47 @@ jobs:
...
 
   test_all_features:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+
+      - name: Start RabbitMQ server
+        run: docker-compose up -d
```
Comment (gftea) [inline .github/workflows/rust.yml:61]: you can call the updated script like below
```suggestion
        run: /start_rabbitmq.sh
```

File: .github/workflows/codecov.yml (hunk 1 above)
Comment (gftea) [inline .github/workflows/codecov.yml:46]:
```suggestion
        run: ./start_rabbitmq.sh
```

File: .github/workflows/rust.yml (hunk 2 above)
Comment (jacobmiller22) [reply .github/workflows/rust.yml:61]: Oh I see. It was only using the container before.

**Review by gftea — APPROVED** [review]
LGTM

**Review by gftea — APPROVED** [review]
LGTM

**Pull request merged**